tests/common-tests.h

```diff
@@ -537,7 +537,7 @@ TEST_CASE_METHOD(common_tests, "Exception on not connected", "[core][exception]"
     CHECK_THROWS_AS(sql.get_last_insert_id(s, l), soci_error);
 }
 
-TEST_CASE_METHOD(common_tests, "Basic functionality")
+TEST_CASE_METHOD(common_tests, "Basic functionality", "basics")
```

Comment:
Hmm, isn't the third argument of [TEST_CASE_METHOD](https://github.com/philsquared/Catch/blob/master/docs/test-fixtures.md) supposed to be a tag?

Aren't [Catch tags](https://github.com/philsquared/Catch/blob/master/docs/test-cases-and-sections.md) supposed to be in form of `["tag1", "tag2"]`?